tests/databrowser.js

```diff
@@ -157,40 +156,13 @@ define([
                 });
             });
 
-            it("should add recursive ref on drag/drop", function(done) {
```

Comment:
Ah I see. I suppose they are still representable, but I didn't want to hardcode caes/parent/grandparent in vellum, so I made a generic flatten everything in this list method. It should be easy enough to change that if we decide recursive relationships are something we want hq to support later (all that would need to happen is to remove `flattenNode`)